crates/router/src/routes/payouts.rs

```diff
@@ -131,6 +132,62 @@ pub async fn payouts_update(
     ))
     .await
 }
+
+/// Payouts - Confirm
+#[utoipa::path(
+    post,
+    path = "/payouts/{payout_id}/confirm",
+    params(
+        ("payout_id" = String, Path, description = "The identifier for payout]")
+    ),
+    request_body=PayoutCreateRequest,
+    responses(
+        (status = 200, description = "Payout updated", body = PayoutCreateResponse),
+        (status = 400, description = "Missing Mandatory fields")
+    ),
+    tag = "Payouts",
+    operation_id = "Confirm a Payout",
+    security(("api_key" = []))
+)]
```

Comment:
I don't think we are using these anymore, can you check this with @Narayanbhat166